### PR TITLE
Added a new command "deleteCategory"

### PR DESCRIPTION
**Pull request by NusMinato**
Added a new command "deleteCategory", allowing the user to delete a category of recorded person.

Example: deleteCategory 1 c/Email

### CODE REVIEW

**Comment by Cedricaca** [comment]
LGTM! Nice job bro

like overall its so good bro idk

like seriously this code is so good with no bugs at all

is this enough words prof

**Comment by Xuyan0518** [comment]
Overall looks good to me. It achieves the desired features. Thanks for the implementation!

**Comment by ChenXy128** [comment]
The code quality is good, easy to read and straightforward. Do rmb to add in javaDoc :)

File: src/main/java/seedu/address/logic/commands/DeleteCategoryCommand.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    /**
+     * @param targetIndex of the person in the filtered person list to edit
+     * @param category    the category needed to be deleted
+     */
+    public DeleteCategoryCommand(Index targetIndex, String category) {
+        this.targetIndex = targetIndex;
+        this.category = category;
+    }
+
+    @Override
```
Comment (HongRay) [inline src/main/java/seedu/address/logic/commands/DeleteCategoryCommand.java:48]: Good method of abstraction, however, please add some description as to what it does.

File: src/main/java/seedu/address/logic/parser/DeleteCategoryCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ */
+public class DeleteCategoryCommandParser implements Parser<DeleteCategoryCommand> {
+    /**
+     * Creates the deletecategorycommand
+     *
+     * @param args our arguments
+     * @return the command
+     * @throws ParseException if no index
+     */
+    @Override
```
Comment (HongRay) [inline src/main/java/seedu/address/logic/parser/DeleteCategoryCommandParser.java:23]: Try to give some description of what your code does for us to understand easily.

**Review by HongRay — COMMENTED** [review]
Overall great code. Nice implementation of delete command. Awesome job 😄

**Comment by HongRay** [comment]
Overall great code. Nice implementation of delete command. Awesome job 😄
However, try to give some description of what your code does for us to understand easily.
Nice choice of making a new class and using good coding standards like abstraction to make your code easier to understand.
I will need to do the same as you hahaha 😆.

**Pull request merged**